src/test/java/org/cactoos/iterator/NoNullsTest.java

```diff
@@ -50,8 +54,15 @@
     public void nextThrowsErrorIfNull() {
         this.exception.expect(IllegalStateException.class);
         this.exception.expectMessage(
-            new StringContains(
-                "Item #0 of org.cactoos.iterator.NoNullsTest"
+            new AllOf<>(
+                new IterableOf<Matcher<? super String>>(
```

Comment:
@mazdack is the explicit typing really needed?

Reply:
yes, otherwise we get a warning

unchecked generic array creation for varargs parameter of type...